asvdb/asvdb.py

```diff
@@ -471,7 +483,10 @@ def __assertDbDirExists(self):
         # FIXME: update for support S3 - this method should return True if
         # self.dbDir is a valid S3 URL or a valid path on disk.
         if self.__isS3URL(self.dbDir):
-            raise NotImplementedError
+            try:
+                self.s3Resource.Object(self.bucketName, self.bucketKey).load()
+            except botocore.exceptions.ClientError as e:
+                raise Exception(e)
```

Comment:
I'm curious why this try/except block is here (and elsewhere), since it's just catching and re-raising the same thing. Can these try/except blocks be removed, or are needed for something?